content/en/profiler/enabling/java.md

```diff
@@ -249,6 +249,8 @@ or:
 ```
 -Ddd.profiling.enabled.events=jdk.ObjectAllocationInNewTLAB,jdk.ObjectAllocationOutsideTLAB
 ```
+
+This is not supported for Oracle JDK 8 and IBM J9 JDK 8. A paid license version of Oracle JDK 8 enables this capability through an upgrade in JFR events.
```

Comment:
I'm not sure the "paid license" for Oracle JDK applies to this event; I believe the "paid license" refers to using any JFR-related features in Oracle JDK 8.

(And I'm not sure if the events above exist in Oracle JDK 8...)

Reply:
Also: maybe perhaps move this to the top, with the rest of the content?